devops/big-dipper/install.sh

```diff
@@ -5,8 +5,10 @@ set -euo pipefail
 apt update
 apt install -y docker.io docker-compose jq
 
+docker-compose pull
```

Comment:
That's good.

You can also use `ExecStartPre=docker-compose pull` inside the service file.

Whichever you like

Reply:
That command is already in the services file. I need to debug it and spend some time. Simple `docker-compose pull` just works.